### PR TITLE
Support rsqrt_p

### PR DESCRIPTION
**Pull request by JiabinYang**
### PR types

Others
### PR changes

Others
### Describe

This PR support rsqrt_p in incubate

### CODE REVIEW

File: paddle/fluid/operators/prim_ops/rsqrt_p_op.cc [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  RsqrtPrimOp(const std::string &type,
+              const framework::VariableNameMap &inputs,
+              const framework::VariableNameMap &outputs,
+              const framework::AttributeMap &attrs)
+      : framework::OperatorBase(type, inputs, outputs, attrs) {}
+  void RunImpl(const framework::Scope &scope,
+               const platform::Place &dev_place) const override {
+    PADDLE_THROW(platform::errors::Unimplemented(
+        "Prim operator sqrt_p should not be excuted directly"));
+  }
```
Comment (cxxly) [inline paddle/fluid/operators/prim_ops/rsqrt_p_op.cc:38]: rsqrt_p

File: paddle/fluid/operators/prim_ops/rsqrt_p_op.cc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+};
+
+class RsqrtPrimOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  void Make() override {
+    AddInput("X", "(Tensor), The input tensor of sqrt_p op.");
+    AddOutput("Y", "(Tensor), The output tensor of sqrt_p op.");
+    AddComment(R"DOC(
+Autograd primitive sqrt_p operator.
```
Comment (cxxly) [inline paddle/fluid/operators/prim_ops/rsqrt_p_op.cc]: rsqrt_p

File: python/paddle/incubate/autograd/primrules.py [hunk 3]
```diff
@@ -969,6 +979,17 @@ def cast_jvp(op, x_dot):
     return primops.cast(x_dot, y.dtype)
 
 
+@REGISTER_JVP('rsqrt_p')
+def rsqrt_jvp(op, x_dot):
+    if x_dot is None:
+        return None
+    y = op_position_output(op)
+    x = op_position_inputs(op)
+    c2 = fill_const(value=-2.0, shape=y.shape, dtype=y.dtype)
```
Comment (cxxly) [inline python/paddle/incubate/autograd/primrules.py:988]: op_position_inputs返回元组，需要解包，  x, = op_position_inputs(op)

**Comment by cxxly** [comment]
需要增加一阶及高阶微分测试用例，具体可以参考test_primapi `sin`

File: paddle/fluid/operators/prim_ops/rsqrt_p_op.cc (hunk 1 above)
Comment (JiabinYang) [reply paddle/fluid/operators/prim_ops/rsqrt_p_op.cc:38]: done

File: paddle/fluid/operators/prim_ops/rsqrt_p_op.cc (hunk 2 above)
Comment (JiabinYang) [reply paddle/fluid/operators/prim_ops/rsqrt_p_op.cc]: done

File: python/paddle/incubate/autograd/primrules.py (hunk 3 above)
Comment (JiabinYang) [reply python/paddle/incubate/autograd/primrules.py:988]: Not really rsqrt return just one element

**Comment by JiabinYang** [comment]
> 需要增加一阶及高阶微分测试用例，具体可以参考test_primapi `sin`

done

**Review by cxxly — APPROVED** [review]
LGTM

**Review by phlrain — APPROVED** [review]

**Pull request merged**